### DISCUSSION TITLE
Unknown Protocol - Response bytes

### DISCUSSION THREAD

**Discussion by ahgit1** [zeek/zeek#1292] [Ideas]
Is it possible to log the first response bytes as well as the first originating bytes in the new unknown _protocol script as I think this could be quite useful ? Also should the id and uid fields be added to the log to tie up with the conn record ?

**Comment by timwoj** [comment]
These reports happen during the packet analysis phase. That's before the connection record is built for the incoming packet, so we wouldn't have that information to log.
